openedx/core/djangoapps/crawlers/models.py

```diff
@@ -0,0 +1,45 @@
+"""
+This module handles the detection of crawlers, so that we can handle them
+appropriately in other parts of the code.
+"""
+from django.db import models
+
+from config_models.models import ConfigurationModel
+
+
+class CrawlersConfig(ConfigurationModel):
+    """Configuration for the crawlers django app."""
+    known_user_agents = models.TextField(
+        blank=True,
+        help_text="A comma-separated list of known crawler user agents.",
+        default='edX-downloader',
```

Comment:
Yeah. The crawler in question is configurable to run against other sites as well, and is used that way.